src/ui_kit/src/input/styles.scss

```diff
@@ -1,23 +1,40 @@
-// Inputs
+.input-container {
+  position: relative;
 
-.input {
-  min-width: 100px;
-  width: 100%;
-  height: 40px;
-  padding-left: 10px;
-  padding-right: 10px;
-  color: var(--theme-text);
-  border-radius: 4px;
-  border: none;
-  background: var(--theme-foreground);
-  box-sizing: border-box;
-  border: 1px solid transparent;
-  transition: 0.2s;
-  margin: 1px;
-}
+  .input {
+    min-width: 100px;
+    width: 100%;
+    height: 40px;
+    padding: 0 10px;
+    color: var(--theme-text);
+    border-radius: 4px;
+    border: none;
+    background: var(--theme-foreground);
+    box-sizing: border-box;
+    border: 1px solid transparent;
+    transition: 0.2s;
+
+    &:focus {
+      outline: none;
+      border: 1px solid var(--theme-primary);
+    }
+  }
+
+  > svg {
+    position: absolute;
+    z-index: 2;
+    stroke: var(--theme-placeholder);
+    top: 11px;
+    left: 10px;
+    fill: transparent;
+    pointer-events: none;
+
+    &:focus {
+      stroke: var(--theme-primary);
+    }
 
-.input:focus {
-  outline: none;
-  border: 1px solid var(--theme-primary);
-  margin: 1px;
+    + .input {
+      padding-left: 35px;
```

Comment:
is this something that only happens when the input has an icon, but has normal padding when it doesn't?

Reply:
Yep :)